disktools.py

```diff
@@ -1267,3 +1267,60 @@ def getMdNodes():
     except IOError:
         pass
     return nodes
+
+
+def readCmdlineMounts():
+    class Mount:
+        def __init__(self, dev, mountpoint, options=None, fstype=None):
+            self.dev = dev
+            self.mountpoint = mountpoint
+            self.options = options
+            self.fstype = fstype
+
+        def __repr__(self):
+            return "Mount(%s, %s, %s, %s)" % (self.dev, self.mountpoint, self.fstype, self.options)
+
+        def mount(self):
+            util.mount(self.dev, self.mountpoint,
+                  self.options if self.options is None else [self.options], self.fstype)
```

Comment:
It might be nicer if you update the mount() util function to handle options as either a list of a string.